docs/INSTALL_VSCODE.md

```diff
@@ -0,0 +1,40 @@
+# Install instruction when using the remote container feature of vscode
+
+The remote container feature allows to run the editors backend insight a docker container which is all setup for development.  
+
+## Quickstart:
+Clone this repository:  
+`git clone https://github.com/ghga-de/datameta.git`
```

Comment:
It would be a bit easier to read if the code fragments weren't inline but block style, IMPO